app/controllers/reports/regions_controller.rb

```diff
@@ -132,6 +138,14 @@ def set_force_cache
   end
 
   def find_region
+    if current_admin.feature_enabled?("region_reports")
+      find_region_v2
```

Comment:
When enabled, _all_ reports will be fueled by regions instead of "sources" (like facilities or facility groups), right? That's a pretty significant change, I think. Any extra testing we need to do? Verify that the caching doesn't get in the way, etc?

Reply:
Yup, once the flag is on that is true.  The plan is to ship this soon so I can verify on sandbox and also get some help from others on QA'ing it.  I'll definitely be doing some updates for caching and testing in follow up PRs before we turn the flag on for everyone.

Reply:
Neatneatneat. Sounds like a good plan.